geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/RedisCommandType.java

```diff
@@ -238,6 +240,8 @@
 
   /*************** Keys ******************/
 
+  DUMP(new DumpExecutor(), UNSUPPORTED, new ExactParameterRequirements(2)),
+  RESTORE(new RestoreExecutor(), UNSUPPORTED, new MinimumParameterRequirements(4)),
```

Comment:
Did we decide to mark these as supported?

Reply:
Doh!